spotless-style.xml

```diff
@@ -1,327 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I’d leave the formatter options and add the Spotless Gradle plugin [1] with the Eclipse jdt formatter [2].

[1] https://github.com/diffplug/spotless/tree/main/plugin-gradle#quickstart
[2] https://github.com/diffplug/spotless/tree/main/plugin-gradle#eclipse-jdt